src/fsdocs-tool/InitCommand.fs

```diff
@@ -0,0 +1,90 @@
+namespace fsdocs
+
+open System.IO
+open CommandLine
+
+[<Verb("init", HelpText = "initialize the necessary folder structure and files for creating documentation with fsdocs.")>]
+type InitCommand() =
+
+    let dir = Path.GetDirectoryName(typeof<InitCommand>.Assembly.Location)
+
+    // get template locations for in-package and in-repo files and decide which to use later
+    let inPackageLocations = Common.InPackageLocations(Path.Combine(dir, "..", "..", ".."))
+    let inRepoLocations = Common.InRepoLocations(Path.Combine(dir, "..", "..", "..", "..", ".."))
+
+    [<Option("output",
+             Required = false,
+             Default = "docs",
+             HelpText = "The output path for the documentation folder structure")>]
+    member val output: string = "docs" with get, set
+
+    member this.Execute() =
+
+        let outputPath = Path.GetFullPath(this.output)
+        let repoRoot = Path.GetFullPath(Path.Combine(outputPath, ".."))
+        let initLocations = Common.InRepoLocations(repoRoot)
+
+        let ensureOutputDirs () =
+            [ outputPath; initLocations.docs; initLocations.docs_img ]
+            |> List.iter ensureDirectory
+
+        if inPackageLocations.Exist() then
+            // if the in-package locations exist, this means fsdocs is run from the nuget package.
+            ensureOutputDirs ()
+
+            try
+                [ (inPackageLocations.template_html, initLocations.template_html)
```

Comment:
I'm very opposed that we copy the `_template.html` file by default.
This should only be done when users are interested in creating their custom theme.
Which, in a sense we want to avoid. 90% of our user base should be able to use or customize the modern theme instead of writing one from scratch.

So, I would not include `_template.html` by default.

Reply:
Sorry I think i do not really get the difference - when the user does not put a `_template.html` file into their docs folder, isn't the file i am copying here the exact one that will be used by the tool per default anyways? 

My rationale would be that you will need a template file once you want to make even slight changes to the layout, but want to keep most of the default file. Where would i get that file other than looking up the file in the FSharp.Formatting repo?

Reply:
On the other hand, this could be circumvented by putting a good descriptive interactive text on the setup

Reply:
Oh i think i get the issue. If we update the file for a new version of the tool, the repos that use the copied template file will have an outdated version. So i agree this should be opt-in

Reply:
Yes, indeed. If the user doesn't have a `_template.html`, the one from NuGet is used.
Which will stay updated if they bump the version.